node/pkg/watchers/sui/watcher.go

```diff
@@ -5,6 +5,9 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"io"
+	"net/http"
+	"strings"
 	"time"
 
 	"encoding/base64"
```

Comment:
We're using "github.com/gorilla/websocket" in all the other watchers so we should stay consistent.  I think the API is basically the same.